kolibri/core/assets/src/core-app/apiSpec.js

```diff
@@ -210,6 +211,9 @@ export default {
   },
   resources,
   themeConfig,
+  machines: {
+    importLodUsersMachine,
```

Comment:
I think I'd rather put this in `kolibri-common` rather than add it to the core API spec.